Source/System.Management/Pash/ParserIntrinsics/AstBuilder.cs

```diff
@@ -424,11 +424,20 @@ private StatementAst BuildBreakStatementAst(ParseTreeNode parseTreeNode)
 
         StatementAst BuildTryStatementAst(ParseTreeNode parseTreeNode)
         {
+            var subClauses = parseTreeNode.ChildNodes[0].ChildNodes;
+            var catchClauses = subClauses[2];
```

Comment:
According to the grammar, the 3rd element of the specific parseTree is either a "finally_clause" or "catch_clauses" sub tree. The latter one has multiple childs representing multiple cacth clauses.

The following functions care about finding the concrete catch clauses in the "sub tree".
However, multiple catch clauses are not yet evaluated correctly, see #414.